README.md

```diff
@@ -1,33 +1,48 @@
+
 [![ko-fi](https://img.shields.io/badge/Support%20me%20on-Ko--fi-FF5E5B.svg?logo=ko-fi&style=flat-square)](https://ko-fi.com/mastercoms)
 [![Liberapay](https://img.shields.io/liberapay/receives/mastercoms.svg?logo=liberapay&style=flat-square)](https://liberapay.com/mastercoms/)
 [![Steam donate](https://img.shields.io/badge/Donate%20via-Steam-00adee.svg?style=flat-square&logo=steam)](https://steamcommunity.com/tradeoffer/new/?partner=85845165&token=M9cQHh8N)
 [![Join the Discord chat](https://img.shields.io/badge/Discord-%23comtress--client-7289da.svg?style=flat-square&logo=discord)](https://discord.gg/CuPb2zV)
 
 
-# Team Comtress 2
+# Welcome to Team Comtress 2!
+Before sharing your ideas and feedback, it's important to have a good understanding of what this project is.
+### What *is* TC2?
+Team Comtress 2 is a fork of an older version of Team Fortress 2, with a community developer team fixing bugs, improving performance, and adding quality of life features, with the goal of having those changes pulled upstream by Valve to the modern game.

```

Comment:
You should just say it's different game tbh.

Reply:
What would be the best way to phrase this?